.editorconfig

```diff
@@ -9,7 +9,7 @@ root = true
 
 # Change these settings to your own preference
 indent_style = space
-indent_size = 4
+indent_size = 2
 
```

Comment:
Id rather not do all of it right now as it makes it even harder to review. Ill do it immediately after this PR.